src/components/Box/Box.tsx

```diff
@@ -16,8 +16,18 @@ export type ResponsiveProp = string | number | Array<string> | Array<number>
 export type Overflow = 'visible' | 'hidden' | 'scroll' | 'auto'
 export type ResponsiveOverflow = Overflow | Array<Overflow>
 
+type Elements = React.HTMLAttributes<HTMLElement> &
+  React.AnchorHTMLAttributes<HTMLAnchorElement> &
+  React.AudioHTMLAttributes<HTMLAudioElement> &
+  React.ButtonHTMLAttributes<HTMLButtonElement> &
+  React.ImgHTMLAttributes<HTMLImageElement> &
+  React.InputHTMLAttributes<HTMLInputElement> &
+  React.LabelHTMLAttributes<HTMLLabelElement> &
+  React.TextareaHTMLAttributes<HTMLTextAreaElement> &
+  React.VideoHTMLAttributes<HTMLVideoElement>
+
 export interface BoxProps
-  extends Omit<React.HTMLAttributes<HTMLElement>, 'color'> {
+  extends Omit<Elements, 'color' | 'height' | 'width' | 'size'> {
```

Comment:
A few new typescript errors popped up with this change.

Grid.tsx can be fixed with this change
```js
export interface GridProps extends Omit<BoxProps, 'columns' | 'rows'> {
  columns?: ResponsiveString
  rows?: ResponsiveString
}
```

Modal.tsx can be fixed with this change
```js
export const Modal = forwardRef(
  (
    { children, ...props }: DialogProps & BoxProps,
    ref: React.Ref<HTMLDivElement>
  ) => {
...

export const ModalContent = forwardRef(
  (
    { children, ...props }: DialogContentProps & BoxProps,
    ref: React.Ref<HTMLDivElement>
  ) => {
```

Then in Radio.tsx and Checkbox.tsx I get an error on `type` prop. Not sure how to fix yet, but I did notice the other `// @ts-ignore` in those files can be removed now.